### PR TITLE
Hardening/refactor const reference

### PR DESCRIPTION
**Pull request by crbrox**

### CODE REVIEW

File: src/lib/ngsi/AttributeList.h [hunk 1]
```diff
@@ -42,7 +42,7 @@ typedef struct AttributeList
 {
   std::vector<std::string>  attributeV;
 
-  void         fill(std::vector<std::string>);
+  void         fill(const std::vector<std::string>&);
```
Comment (kzangeli) [inline src/lib/ngsi/AttributeList.h]: The name of the parameter might not be mandatory but it's good to have it ...
I think ...
This is really a question for the style guide. Do we want to enforce 'named parameters' always ?
Opinions, please ...

Comment (crbrox) [inline src/lib/ngsi/AttributeList.h]: Fixed 50e1659

**Comment by fgalan** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**